### PR TITLE
describe exclusion NGRX changes

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3951

### Change description ###

Describe the exclusion-Continue and Cancel- Roles and Access tab

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
Not too familiar with NgRx but functionally all looks good and tests are helpful

**Review by kuda1992 — APPROVED** [review]
Looks good

File: src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.spec.ts [hunk 1]
```diff
@@ -1,33 +1,46 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { provideMockStore } from '@ngrx/store/testing';
+import { async } from '@angular/core/testing';
 import { DescribeExclusionComponent } from './describe-exclusion.component';
+import { Validators } from '@angular/forms';
+import { ExclusionNavigationEvent, ExclusionState } from '../../../models';
+import { UpdateDescribeExclusionText } from 'src/role-access/store';
```
Comment (rayliangatos) [inline src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.spec.ts:5]: src

Comment (udaydenduluri33) [reply src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.spec.ts]: Fixed the route now

File: src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.html [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+  </span>
+  <fieldset class="govuk-fieldset">
+    <textarea
+      [ngClass]="{'govuk-textarea--error': !formGroup.get(controlName).valid && formGroup.get(controlName).touched && submitted}"
+      class="govuk-textarea govuk-!-width-one-half govuk-!-margin-bottom-0"
+      id="exclusion-description"
+      formControlName="text"
+      name="exclusion-description"
+      rows="5"
+      aria-describedby="exclusion-description"
```
Comment (rayliangatos) [inline src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.html:21]: it seems this aria-describedby not set correclty, it's binding itself but not other element.
see usage on:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute, it seems it's more proper to use aria-label="enter exclusion description" here.

Comment (udaydenduluri33) [reply src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.html]: @rayliangatos  fixed it now

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**